src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.motorcontrol.MotorUtil;
+
+import static frc.robot.Constants.RollerConstants.*;
+
+public class RollerSubsystem extends SubsystemBase{
+    
+    private final WPI_TalonSRX leftbeak = MotorUtil.createTalonSRX(LEFTID);
+    private final WPI_TalonSRX rightbeak = MotorUtil.createTalonSRX(RIGHTID);
+    
+    //limit switch
+    private final DigitalInput crimitswitch = new DigitalInput(0);
+    //timer
+    public Timer timere = new Timer();
+    //roller state 
+    public String rollstate;
```

Comment:
if we use a Java Enum instead we can make sure the value is one of the few that we expect.

that being said, don't we want to give drivers the option to control how fast it's intaking? i think having a double intakePower as this state wouldn't be too unreasonable